src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -735,7 +735,32 @@ def minimize_crash(self,
                      timeout,
                      artifact_prefix=None,
                      additional_args=None):
-    return new_process.ProcessResult()
+    self._test_ssh()
+
+    additional_args = copy.copy(additional_args)
+    if additional_args is None:
+      additional_args = []
+    additional_args.append('-minimize_crash=1')
+
+    # We need to push the testcase to the device and pass in the name.
+    testcase_path_name = os.path.basename(os.path.normpath(testcase_path))
+    self.device.store(testcase_path, self.fuzzer.data_path())
+
+    return_code = self.fuzzer.start(additional_args +
```

Comment:
We also need to pass a timeout. See LibFuzzerCommon.minimize_crash for an example. You should base the arguments you pass off that. Also, I just realised you need to add this to merge() as well. (You have a TODO there :) )

Reply:
fixed it in `merge`, as well as here